shared/src/main/scala/com/nawforce/pkgforce/diagnostics/IssuesManager.scala

```diff
@@ -106,29 +130,11 @@ class IssuesManager extends IssuesCollection with IssueLogger {
   }
 
   override def issuesForFile(path: String): Array[APIIssue] = {
-    issuesForFileInternal(Path(path)).toArray
+    issuesForFilesInternal(Array(Path(path)), includeWarnings = true, maxIssuesPerFile = 0).toArray
```

Comment:
Perhaps we can default these args that are always the same, I commented on this in a different file as well.